draft-yasskin-wpack-bundled-exchanges.md

```diff
@@ -92,8 +93,8 @@ Standard ({{INFRA}}).
 
 # Semantics {#semantics}
 
-A bundle is logically a set of HTTP exchanges, with a URL identifying the
-manifest(s) of the bundle itself.
+A bundle is logically a set of HTTP exchanges, with a primary URL in the bundle
+and an optional URL identifying the manifest(s) of the bundle itself.
```

Comment:
No need to mention manifest here. How about this?
```
A bundle is logically a set of HTTP exchanges, with a URL identifying the
primary resource of the bundle.
```

Reply:
done